lib/moonshot/commands/new.rb

```diff
@@ -0,0 +1,82 @@
+module Moonshot
+  module Commands
+    class New < Moonshot::Command
+      self.usage = 'new [options]'
+      self.description = 'Creates a new Moonshot project.'
+
+      DEFAULT_DIRECTORY = File.join(__dir__, '..', '..', 'default').freeze
+
+      def execute
+        warn 'Looks like your project is already set up!'
+      end
+
+      class << self
+        def run!(application_name)
+          @application_name = application_name
+
+          create_project_dir
+          copy_defaults
+          create_file(parameter_path)
+          create_file(template_path)
+          fill_moonfile
```

Comment:
Can we add some output here that says something like:

---
Your application is configured, the following changes have been made to your project directory:
- Created a Moonfile.rb where you can configure your project.
- Created moonshot/plugins where you can add hooks to core Moonshot actions.
- Created moonshot/cli_extensions where you can create project-specific commands.

You will also need to ensure your amazon account is configured for CodeDeploy, by creating a role that allows deployments. See: [link to moonshot documentation].